jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/grid/dash/DataSetTaskAdminGridViewImpl.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.workbench.ht.client.editors.taskslist.grid.dash;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import javax.enterprise.context.Dependent;
+
+import org.gwtbootstrap3.client.ui.Button;
+import org.jbpm.workbench.df.client.filter.FilterSettings;
+import org.jbpm.workbench.df.client.filter.FilterSettingsBuilderHelper;
+import org.jbpm.workbench.common.client.util.TaskUtils;
+import org.uberfire.ext.services.shared.preferences.GridGlobalPreferences;
+import org.uberfire.ext.widgets.common.client.tables.popup.NewTabFilterPopup;
+
+import static org.dashbuilder.dataset.filter.FilterFactory.*;
+import static org.jbpm.workbench.ht.model.TaskDataSetConstants.*;
+
+@Dependent
+public class DataSetTaskAdminGridViewImpl extends BaseDataSetTasksListGridViewImpl {
+
+    private static final String DATASET_TASK_LIST_PREFIX = "DataSetTaskAdminGrid";
+    
+    @Override
+    public FilterSettings createTableSettingsPrototype() {
+        FilterSettingsBuilderHelper builder = FilterSettingsBuilderHelper.init();
```

Comment:
The only different between the two views is the dataset name, perhaps you could move this all to the parent class and only get the dataset name.